google/cloud/aiplatform/jobs.py

```diff
@@ -312,6 +316,22 @@ def create(
                 The maximum number of machine replicas the batch operation may
                 be scaled to. Only used if `machine_type` is set.
                 Default is 10.
+            explanation_metadata (aiplatform.ExplanationMetadata):
```

Comment:
Thanks for catching this, also allowed `explanation_metadata` to be passed independently of `explanation_parameters` since all these fields are for optional overriding.